### DISCUSSION TITLE
Enhancement: Printable chats

### DISCUSSION THREAD

**Discussion by edemaine** [danny-avila/LibreChat#2483] [Feature Requests & Suggestions]
### What features would you like to see added?

Currently, if I try to print a longer chat, I only get the first page, because of some layout/overflow rules. Print CSS should disable this layout (and probably hide sidebars, though I can do that manually) so that I get the entire printout.

### More details

Context: I'd like to share an entire chat session with someone. Printing to PDF seems like an easy way.

### Which components are impacted by your request?

UI

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
There is an export conversation option when you have an active conversation open

![image](https://github.com/danny-avila/LibreChat/assets/110412045/dc52a517-a96f-4b68-ab0b-b38f52713d1f)

hope to make this feature more obvious, as you can get nicer print this way
